frontend/src/features/properties/map/acquisition/add/AcquisitionTeam.tsx

```diff
@@ -0,0 +1,96 @@
+import { LinkButton } from 'components/common/buttons';
+import { Button } from 'components/common/buttons/Button';
+import { Select } from 'components/common/form';
+import { ContactInput } from 'components/common/form/ContactInput';
+import { ContactManagerModal } from 'components/contact/ContactManagerModal';
+import * as API from 'constants/API';
+import { FieldArray, FormikProps, useFormikContext } from 'formik';
+import useLookupCodeHelpers from 'hooks/useLookupCodeHelpers';
+import { IContactSearchResult } from 'interfaces/IContactSearchResult';
+import { useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { MdClose } from 'react-icons/md';
+
+import { AcquisitionForm, AcquisitionTeamForm } from './models';
+
+interface AcquisitionTeamProp {
+  formikProps: FormikProps<AcquisitionForm>;
+}
+export const AcquisitionTeam: React.FunctionComponent<AcquisitionTeamProp> = ({ formikProps }) => {
```

Comment:
Given this is a form I believe AcquisitionTeamForm would be better

Reply:
Changed it to AddAcquisitionTeamForm